src/vis/vis.js

```diff
@@ -33,13 +40,31 @@ var VisModel = Backbone.Model.extend({
     this.overlaysCollection = new Backbone.Collection();
   },
 
+  setOk: function () {
+    this.trigger('ok');
```

Comment:
(this is general feedback, I write here instead of the PR to have a threaded conversation about this particular issue):

It's not clear to me what _ok_ really means, it feels a bit too generic/unclear/ambiguous. What do you think about clarifying it, depending on the model?  

Also, why not remove the duplication of state data (event+state attr), in favor of just having one data point? This also have some benefits:
- better model the intent, that these "states" are really mutually exclusive
- get rid of manual triggers
- be able to get state after (last) event happened

```js
vis.on(`change:state`, …
vis.get('state'); // => 'error' 
```